qa/tasks/cephadm.py

```diff
@@ -161,7 +174,17 @@ def download_cephadm(ctx, config, ref):
             '--fsid', ctx.ceph[cluster_name].fsid,
             '--force',
         ])
-
+        if ctx.config.get('redhat'):
```

Comment:
since you're letting the install task handle the install, it should be responsible for the uninstall as well - no need for this in the cephadm task

Reply:
I Agree.